src/main/java/frc/robot/Constants.java

```diff
@@ -85,9 +85,9 @@
       public static final class ShooterConstants {
         public static final int kShooterMotorPort = 30;
         public static final double kShooterSpeedSlow = 0.6;
-        public static final double kShooterFastRPM = 4000;
```

Comment:
Did you really intend to lower the shooter variables? Are these values based on your testing?

Reply:
I did not mean to lower the values. My dad was playing around with the constants and wanted me to set them to what he had. However, we do need to get the constants from whatever branch he put them in as those were the constants that were working most reliably yesterday.

Reply:
I fixed the motor speed to match main branch as that was what they were testing yesterday.